blueocean-dashboard/src/main/less/core.less

```diff
@@ -390,9 +390,11 @@ table.highlight {
     width: 100%;
     height: 100%;
     display: block;
-    padding: 0.5em 0;
+    padding: 0.5em 0.8em 0.5em 0;
     text-decoration: none;
     color: inherit;
+    line-height: 22px; // force height to fill space to ensure clicks are "captured"
```

Comment:
I don't really like this approach but the TR elements were introducing minor space above/below each TD. I tried positioning them absolutely but then there were vertical alignment issues. Maybe `flexbox` and `alignItems` would handle that better.